.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "interactive"
+}
```

Comment:
Please remove this and put in the gitignore

Reply:
Thanks for responding. I'm happy to maintain this adapter if your team is not using it anymore. I am curious as to why you are not and what you switched to as obviously I'm planning on using it for a project.  I thought it was appropriate to change the name but I'm happy to change it back if it proves problematic. I've made the changes you've requested see the latest commit.